docs/sample_inventories/migration_sample.yml

```diff
@@ -0,0 +1,55 @@
+---
+##
+## The following is an example inventory file for Migration of ZK cluster to Kraft cluster
+## steps to run:
+## 1. Comment out "migration_enabled: true" and kafka_controller section from inventory file
+## 2. Run command "ansible-playbook -i <inv.yml> confluent.platform.all"
+## 3. ZK cluster is up and running
+## 4. uncomment "migration_enabled: true" and kafka_controller section. Note that enabling jolokia in kafka_controller is a prerequisite.
```

Comment:
will add a task to check jolokia path is available for air-gapped env or this url should be accessible as prereq

Reply:
since writing and testing pre-check playbook will take some time, will it be good if I include that in the next PR (RBAC)? Will link that PR here for future reference

Reply:
added jolokia tests in the pre-check playbook [here](https://github.com/confluentinc/cp-ansible/blob/821dababaeabb2222c0df22bce1b545ef2066c51/playbooks/migration_precheck.yml#L167)